lib/config/rules/jsx-a11y.js

```diff
@@ -19,28 +25,42 @@ module.exports = {
   'jsx-a11y/href-no-hash': 'error',
   // Enforce <html> element has lang prop.
   'jsx-a11y/html-has-lang': 'error',
-  // Enforce that <img> JSX elements use the alt prop.
-  'jsx-a11y/img-has-alt': 'error',
+  // Enforce iframe elements have a title attribute.
+  'jsx-a11y/iframe-has-title': 'error',
   // Enforce <img> alt prop does not contain the word "image", "picture", or "photo".
   'jsx-a11y/img-redundant-alt': 'warn',
+  // Enforce that elements with interactive handlers like onClick must be focusable.
+  'jsx-a11y/interactive-supports-focus': 'error',
   // Enforce that <label> elements have the htmlFor prop.
   'jsx-a11y/label-has-for': 'error',
+  // Enforces that <audio> and <video> elements must have a <track> for captions. <FEEDBACK>
+  'jsx-a11y/media-has-caption': 'off',
   // Enforce lang attribute has a valid value.
   'jsx-a11y/lang': 'error',
   // Enforce that onMouseOver/onMouseOut are accompanied by onFocus/onBlur for keyboard-only users.
   'jsx-a11y/mouse-events-have-key-events': 'error',
   // Enforce that the accessKey prop is not used on any element to avoid complications with keyboard commands used by a screenreader.
   'jsx-a11y/no-access-key': 'error',
+  // Enforce autoFocus prop is not used. <FEEDBACK>
+  'jsx-a11y/no-autofocus': 'off',
+  // Enforce distracting elements are not used. <FEEDBACK>
+  'jsx-a11y/no-distracting-elements': 'off',
   // Enforce <marquee> elements are not used.
   'jsx-a11y/no-marquee': 'error',
+  // Interactive elements should not be assigned non-interactive roles.
+  'jsx-a11y/no-interactive-element-to-noninteractive-role': 'off',
+  // Non-interactive elements should not be assigned mouse or keyboard event listeners.
+  'jsx-a11y/no-noninteractive-element-interactions': 'error',
+  // Non-interactive elements should not be assigned interactive roles. <FEEDBACK>
+  'jsx-a11y/no-noninteractive-element-to-interactive-role': 'off',
+  // tabIndex should only be declared on interactive elements. <FEEDBACK>
+  'jsx-a11y/no-noninteractive-tabindex': 'error',
```

Comment:
Yes, keep this on.  It's not a _great_ metric for jsx rules, but a browse through Admin code suggests that this would provide useful feedback.